app/assets/stylesheets/stylesheets/lesson-content.css

```diff
@@ -114,11 +114,11 @@
     @apply hover:no-underline;
   }
 
-  .lesson-content h3 > a {
+  .lesson-content :is(h3, h4) > a {
```

Comment:
Really nice use case for `:is` - I'm pretty sure this is the first time I'm seeing it used in the wild lol